README.md

```diff
@@ -15,7 +15,7 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
 
 ### Related Projects
 
-[Android Specific](https://github.com/realeyes-media/scte35-kotlin)
+[Android Specific](https://github.com/realeyes-media/scte35-android)
```

Comment:
Do we want two sections? "Related Projects" with `scte35-kotlin` & `scte35-swift` plus another section called "Reference Projects" with `scte35-android` & ?

Reply:
@realeyes-mike-patterson  It is only one section which should be called reference projects i will change this

Reply:
Done

Reply:
I was wondering if we wanted to add a section? Because https://github.com/realeyes-media/scte35-swift isn't a reference project. It's a related project that does the same thing in Swift.

Reply:
I think no because it the same code , we did this before i think because we were not able to generate a library to use it

Reply:
Ok, well let's not forget to update scte35-swift once we have a reference iOS project